ui/src/message_popup/popup_question.jsx

```diff
@@ -148,7 +148,6 @@ export default React.createClass({
     return (
       <div>
         <VelocityTransitionGroup leave={{animation: "slideUp"}} runOnMount={true}>
-          {!scaffolding.shouldShowSummary && this.renderQuestion()}
           {!isDoneRevising && scaffolding.shouldShowSummary && this.renderQuestion()}
```

Comment:
@kevinrobinson I actually think this line was needed.
Line 151 renders the question if the summary does not get shown and line 152 renders the question if the summary should be shown but the question is still in progress. 

If you toggle off showing summary, you get what happens below.

line 151 removed vs line 151 included.
<div style="display: flex; flex-direction: row">
<img width="270" alt="screen shot 2016-08-10 at 10 39 41 am" src="https://cloud.githubusercontent.com/assets/1490352/17557898/e7adc37e-5ee6-11e6-8fb0-72f9859b8d23.png">
<img width="270" alt="screen shot 2016-08-10 at 10 40 02 am" src="https://cloud.githubusercontent.com/assets/1490352/17557911/ee338eae-5ee6-11e6-960d-66a9960819fc.png">

</div>